README.md

```diff
@@ -308,6 +308,127 @@ for vehicle in vehicle_path_list:
 
 All of the simulations will be launched in one go. The user can check the status of the simulations and retrieve the results when they are ready. Check the FAQ section for more information on how to do this.
 
+## Task management
+
+
+### Using the `Task` class
+
+As you may have noticed, both the sync and async runs of the scenarios show above return a `Task` object to the user.
+This object provides several methods to manage a task submitted to the **Inductiva API**.
+For instance, you can get information about the task (its current status, when it started, information on the machine it is running, ...), kill it while
+it is still running if you've changed your mind, or download its output files (all of them or only those you need) after it completed.
+
+Check out some examples after submitting an asynchronous simulation:
+
+```python
+# `scenario`` constructed as in the examples above.
+# ... stands for arguments related to each specific scenario.
+task = scenario.simulate(..., run_async=True)
+
+# Kill a task that is running in the API.
+task.kill()
+```
+
+Or if you
+
+```python
+
+task = scenario.simulate(..., run_async=True)
+
+# Get the current status of the task.
+status = task.get_status()
+
+# Block until the results are ready and get an output object.
+# `output` objects vary depending on the scenario, exposing methods to
+# manipulate the results that are relevant for that scenario!
+output = task.get_output()
+
+# Alternatively, download the raw output files!
+
+# If you don't know the files generated by your task, you can get an object
+# representing information on the output files.
+outputs_info = task.get_output_files_info()
+
+full_download_size = outputs_info.size # Size of the full output archive.
+files = outputs_info.contents # List of individual files available.
+
+# Pretty print information on the output archive.
+print(outputs_info)
+# Console logs:
+#
+# Archive size: 1.64 GiB
+# Contents:
+#  Size         Compressed   Name
+#  191 B        96 B         stderr.txt
+#  138.24 KiB   16.73 KiB    stdout.txt
+# ...
+#  3.80 MiB     3.61 MiB     important_file1.txt
+#  1.27 MiB     1.21 MiB     important_file2.txt
+
+
+# Download only those you are interested in:
+output_dir = task.download_outputs(
+    filenames=["important_file1.txt", "important_file2.txt"])
+# Console logs:
+#
+# 100%|██████████| 4.82M/4.82M [00:00<00:00, 273MiB/s]
+
+# Or all the generated files for archival, without extracting the
+# downloaded zip.
+output_dir = task.download_outputs(uncompress=False)
```

Comment:
@luispcunha : please explain @hpenedones the way this download mechanism is working now. Do we still have that crazy bottleneck? Is @hpenedones aware of that?

Reply:
I think he's tagged on the issues but I'll talk to him about it since maybe it was while he was away and he missed it.